hackathon_site/dashboard/frontend/src/slices/event/teamSlice.test.ts

```diff
@@ -74,6 +75,18 @@ describe("Selectors", () => {
         expect(teamCodeSelector(teamState)).toEqual(mockTeam.team_code);
     });
 
+    test("teamSizeSelector returns false", () => {
```

Comment:
In this case, the description should be "returning true", since you use a valid team